zone/mob.h

```diff
@@ -1422,12 +1437,12 @@ class Mob : public Entity {
 	int targeted;
 	bool findable;
 	bool trackable;
-	int32 current_hp;
-	int32 max_hp;
-	int32 base_hp;
-	int32 current_mana;
-	int32 max_mana;
-	int32 hp_regen;
+	int64 current_hp;
+	int64 max_hp;
+	int64 base_hp;
+	int64 current_mana;
+	int64 max_mana;
+	int64 hp_regen;
 	int64 hp_regen_per_second;
 	int32 mana_regen;
 	int32 ooc_regen;
```

Comment:
int64 here?

Reply:
`hp_regen` isn't fully using int64 so I didn't touch OOC fully

Only `hp_regen_per_second` is int64

Reply:
Okay, should mana_regen be int64 there?

Reply:
I thought you were pointing at ooc_regen, I can scoop these up real quick

Reply:
Didn't know if both or just one/other was supposed to be. Thanks. 👍